CMakeLists.txt

```diff
@@ -33,6 +34,9 @@ function(add_security_flags target)
         endif()
     elseif(WIN32)
         target_compile_options( ${target} PRIVATE /GS /DynamicBase)
+        if(BUILD_STATIC)
+            target_compile_options( ${target} PRIVATE "/MT")
```

Comment:
This way is not consistent.
You can build both static/shared library with both static/dynamic MSVC runtime.

BTW, OpenCV itself uses `BUILD_WITH_STATIC_CRT` variable to control that.

Reply:
Yep, but in our project, Inference Engine, will get ade as a third-party libs alone. So there are no flags to control the CRT version. Or, I can just modify the MT/MD option manually?